_articles/touchpad.md

```diff
@@ -21,3 +21,13 @@ Press <kbd>Fn</kbd>+<kbd>F1</kbd> to turn your laptop touchpad on/off.
 To configure your touchpad, press the Super Key <kbd><span class="fl-ubuntu"></span></kbd>, <kbd><span class="fl-pop-key"></span></kbd> then type <u>Mouse & Touchpad</u> and click on the result. The touchpad settings are under the **Touchpad** tab.
 
 ![Mouse Search](/images/touchpad/mouse.png)
+
+<u>GNOME Tweaks</u> has a few other options for the touchpad like disabling it while typing, the acceleration profile and mouse click emulation.
```

Comment:
Now that this a markdown document you don't need to use HTML tags. This can be updated to

```suggestion
_GNOME Tweaks_ has a few other options for the touchpad like disabling it while typing, the acceleration profile and mouse click emulation.
```

Reply:
https://www.markdownguide.org/cheat-sheet/